src/pages/bpd.astro

```diff
@@ -0,0 +1,227 @@
+        successful completion of the cohort program,
+        participants will receive a certificate recognizing
+        their proof-of-work.
+      </p>
+    </div>
+
+    <ul slot="takeaways" class="list-disc">
+      {
+        takeaways.map((takeaway) => (
+          <li class="my-4 lg:text-xl">
+            <span class="font-bold">{takeaway.name}</span> :{" "}
+            {takeaway.desc}
+          </li>
+        ))
+      }
+    </ul>
+  </Description>
+  <CohortDetails>
+    <div slot="prerequisites">
+      <ol class="list-disc p-4 lg:px-8">
+        <li>
+          Basic knowledge of Bitcoin and familiarity with
+          terminal operations
+        </li>
+        <li>Cover the material in Mastering Bitcoin</li>
+        <li>
+          Become comfortable with Unix/Linux command-line
+          terminals
+        </li>
+      </ol>
+      <div
+        class="my-10 grid grid-cols-2 gap-10 lg:grid-cols-4"
+      >
+        {
+          format.map((item) => (
+            <div>
+              <h2 class="font-header text-xl font-bold lg:text-2xl">
+                {item.name}
+              </h2>
+              <p class="lg:text-lg">{item.desc}</p>
+            </div>
+          ))
+        }
+      </div>
+    </div>
+    <div slot="details">
+      <p class="mb-6 font-header lg:text-xl">
+        Cohort level: <span class="text-white"
+          >Intermediate</span
+        >
+      </p>
+      <p class="mb-6 font-header lg:text-xl">
+        The cohort program will comprehensively cover <span
+          class="text-white">chapters 01 to 13</span
+        > of "Learning Bitcoin From Command Line," this
+        provides participants with a solid foundation in
+        Bitcoin operations through the command line.
+      </p>
+      <p class="mb-6 font-header lg:text-xl">
+        After the cohort, participants are encouraged to <span
+          class="text-white"
+          >independently explore chapters 14 to 20</span
+        >
+         , which delve into the advanced functionalities of
+        Bitcoin Core. This will allow them to further
+        explore more advanced topics like programmatically
+        interfacing with bitcoin-cli and the lightning
+        network.
+      </p>
+    </div>
+  </CohortDetails>
+</Layout>
+<Syllabus>
+  <div slot="syllabus">
+    {
+      syllabus.map((item, index) => (
+        <>
+          <div
+            class={`flex items-center justify-center  ${
+              index % 2 ? "bg-peach" : "bg-white"
+            } px-3 py-5`}
+          >
+            <p class="w-1/6 rounded-lg bg-orange py-2 text-center text-white lg:w-1/12">
+              {item.week}
+            </p>
+            <p class="w-4/6 pl-5 text-xl font-bold lg:w-10/12">
+              {item.name}
+            </p>
+            <p class="w-2/6 lg:w-1/12">{item.chNo}</p>
+          </div>
+          <hr class=" border-spacing-12 border-dashed" />
+        </>
+      ))
+    }
+  </div>
```

Comment:
This seems to be the same styling for all pages, can we move it into syllabus component?